xeth/xeth.go

```diff
@@ -379,11 +394,11 @@ func (self *XEth) CurrentBlock() *types.Block {
 }
 
 func (self *XEth) GetBlockReceipts(bhash common.Hash) types.Receipts {
-	return self.backend.BlockProcessor().GetBlockReceipts(bhash)
+	return self.backend.BlockProcessor().GetBlockReceipts(bhash) //ODR
 }
 
-func (self *XEth) GetTxReceipt(txhash common.Hash) *types.Receipt {
-	return core.GetReceipt(self.backend.ChainDb(), txhash)
+func (self *XEth) GetTxReceipt(txhash common.Hash) *types.Receipt { //ODR
+	return core.GetReceipt(self.backend.ChainAccess(), txhash) //TODO
```

Comment:
It is, retrieving individual receipts by tx hash doesn't work yet, only retrieving all receipts from a block by block hash. We have to add some new messages to the LES/1 protocol to cover relaying transactions and checking their status but it's still being figured out.